tagstudio/src/core/library/alchemy/visitors.py

```diff
@@ -16,6 +16,15 @@
 else:
     Library = None  # don't import .library because of circular imports
 
+FILETYPE_EQUIVALENTS = [set(["jpg", "jpeg"])]
```

Comment:
For the JPG set I have a couple of suggested additions: `jpeg_large`/`jpg_large` and `jfif`. As far as I *know*, `jfif` should be the same as `jpeg`.

It might also be easier going forward to break out these sets into something like the `media_types.py` so they'll be easier to find, modify, and use elsewhere if needed.

Reply:
What do you think about adding this to `core/constants.py`? Since this is not specifically SQL related it should go into core anyways so it would fit

Reply:
`core/media_types.py` is not directly related to SQL and in fact predates the SQL migration - It's just a general core library for all file type / extension / media type information.

At first I considered `core/constants.py` but thought that `core/media_types.py` would be a better fit since it's also the file that handles all related media type "constants" and definitions.

Reply:
I don't know how I missed that `media_types.py` already existed lol
Should be fixed now!
(This was a 3AM commit though, maybe read it over :) )